gradle.properties

```diff
@@ -1,7 +1,12 @@
 #Settings to PivotalTracker - UI
-authenticationBaseURL=https://www.pivotaltracker.com
+
+authentication.baseURL=https://www.pivotaltracker.com
```

Comment:
LOL, I wasn't aware that I've changed that line,... Probably I change it by mistake